bigquery/query_join_your_data.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+title: Joining with Your Own Data
+nav_order: 4
+parent: DC to BQ Sample Queries
+grand_parent: BigQuery
+---
+
+# Query Category: Joining with Your Own Data
+
+* Using FIPS code:
+
+Join with Fatal Accidents report from National Highway Traffic Safety Administration to compute counties with highest fatal accidents per capita. Winner is Loving County, TX (the least populated county in main US), followed by Kenedy County, TX.
```

Comment:
Could we link to the analytics hub reference?

Reply:
Done